adoorback/category/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+
+# Create your models here.
+
+from django.conf import settings
+from django.db import models
+from django.db.models import Q
+from safedelete.models import SafeDeleteModel
+from safedelete import SOFT_DELETE_CASCADE
+
+from adoorback.models import AdoorTimestampedModel
+
+class Category(AdoorTimestampedModel, SafeDeleteModel):
+    HIERARCHY_CHOICES = [
+        ('public', 'Public'),
+        ('follower', 'Follower'),
+        ('friend', 'Friend'),
+        ('private', 'Private'),
+    ]
+
+    name = models.CharField(max_length=100)
+    added_users = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name='added_categories', blank=True)
+    sharing_scope = models.CharField(max_length=255)
+
```

Comment:
I just realized, I think you need a field like `user` or `owner` here that stores the owner of the category!

Reply:
Ah yes, I overlooked that - I've added an owner for the Category model too!